### PR TITLE
fix: document for backward compatibility test

### PR DESCRIPTION
**Pull request by cherrycl**
Document and execution file for backward compatibility test.
For more details [edgex-taf-pipeline #108](https://github.com/edgexfoundry/edgex-taf-pipelines/issues/108)

Signed-off-by: Cherry Wang <cherry@iotechsys.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [X] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

## Testing Instructions

See [run-bacward-compatibility-test-on-local.md](https://github.com/edgexfoundry/edgex-taf/compare/main...cherrycl:bct-docs?expand=1#diff-e588c33695a934fa11f7c38bb4f661422d7a547f1102302109218b2f75ed3e32)

### CODE REVIEW

File: docs/run-bacward-compatibility-test-on-local.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+###  Variable configuration
+Export the following variables
+
+```
+export WORK_DIR=/home/$USER/edgex-taf
+```
+
+### Run scripts
+##### Parameters
+- CASE: 1 to 4 (Please see cases detail on [Test Scenarions](./run-bacward-compatibility-test-on-local.md#Test Scenarios)
```
Comment (jumpingliu) [inline docs/run-bacward-compatibility-test-on-local.md:49]:
```suggestion
- CASE: 1 to 4 (Please see cases detail on [Test Scenarions](./run-backward-compatibility-test-on-local.md#Test Scenarios)
```
and the title of run-bacward-compatibility-test-on-local.md.

**Review by jumpingliu — COMMENTED** [review]
Here to vertify that run-bct-test.sh works fine.
Just a minor thing to fix.

File: docs/run-bacward-compatibility-test-on-local.md (hunk 1 above)
Comment (cherrycl) [reply docs/run-bacward-compatibility-test-on-local.md]: Updated.

**Review by jumpingliu — APPROVED** [review]
LGTM

**Pull request merged**